contents/handbook/people/team-structure/platform.md

```diff
@@ -13,62 +13,48 @@ hideAnchor: true
 
 ## Mission
 
-Make using and developing for PostHog as reliable as running water. Wherever you want it.
-
-## Goals 
-
-- We don't lose events
-- Data is as up to date as possible
-- Engineers always be able to ship and build
-- Fail fast. Fix faster.
-- Ship anywhere
-- Stack scales with demand
-- Support Small Teams (and contributors) in building and debugging PostHog
-- Be frugal.
-
-## Responsibilities
-Concrete things we take responsibility over:
-
-- [app.posthog.com](https://app.posthog.com) and its infrastructure
-- On Prem & Single Tenant deployments
-- CI/CD - How we deploy
-- Data infrastructure (ClickHouse, Kafka)
-- Monitoring and Alerting stack
-- Event ingestion
-- Extensibility: Plugin server, plugins, and libraries
-
-## Customer
-
-- Other Small Teams in making sure they have the tools (databases, queues, etc) and the ability to deploy effortlessly that they need to build
-- End users (Both cloud and on-prem teams)
-
-## Output metrics
-
-### VPC
-###### Retention 
-- Metric: Retention
-- Objective: Better than cloud
-### Cloud
-###### Data Loss
-- Metric: Data loss %
-- Objective: < 0.1%
-###### Uptime 
-- Metric: Uptime
-- Objective: > 99.99%
-###### Speed 
-- Metric: Speed
-- Objectives
-  - Event ingestion: TBD
-  - Query response: TBD
-- Overall: We should anticipate increasing demand (either manually or automatically)
-##### Cost 
-- Metric: Infra Costs 
-- Objective: Our costs should grow at a rate that is sublinear relative to scale
-### Dev experience
-##### Dev experience NPS (Infra)
-- Metric: Developer experience (relating to infra) (maybe NPS?)
-- Objective: TBD (maybe NPS?)
+Make deploying, scaling and managing PostHog easy
+
+## Goals & Key Results
+
+* PostHog is the easiest self-hosted product to deploy and scale in the world
+    * Result: Volume of clients requring help @marcushyett-ph
+    * Result: % of successful installs / upgrades @guidoiaquinti
+    * Result: Max size of operating instances @fuziontech
+    * Montior: % of users on latest version of PostHog @tiina303
```

Comment:
Added https://app.posthog.com/insights/DFGqpUoM?insight=TRENDS&events=%5B%7B%22id%22%3A%22instance%20status%20report%22%2C%22math%22%3A%22dau%22%2C%22name%22%3A%22instance%20status%20report%22%2C%22type%22%3A%22events%22%2C%22order%22%3A0%2C%22properties%22%3A%5B%7B%22key%22%3A%22posthog_version%22%2C%22type%22%3A%22event%22%2C%22value%22%3A%221.1.*%22%2C%22operator%22%3A%22not_regex%22%7D%2C%7B%22key%22%3A%22posthog_version%22%2C%22type%22%3A%22event%22%2C%22value%22%3A%221.2.*%22%2C%22operator%22%3A%22not_regex%22%7D%2C%7B%22key%22%3A%22posthog_version%22%2C%22type%22%3A%22event%22%2C%22value%22%3A%22is_set%22%2C%22operator%22%3A%22is_set%22%7D%5D%2C%22custom_name%22%3A%22%3E%3D%201.30%22%7D%2C%7B%22id%22%3A%22instance%20status%20report%22%2C%22math%22%3A%22dau%22%2C%22name%22%3A%22instance%20status%20report%22%2C%22type%22%3A%22events%22%2C%22order%22%3A1%2C%22properties%22%3A%5B%7B%22key%22%3A%22posthog_version%22%2C%22type%22%3A%22event%22%2C%22value%22%3A%22is_set%22%2C%22operator%22%3A%22is_set%22%7D%5D%2C%22custom_name%22%3A%221.2.*%22%7D%2C%7B%22id%22%3A%22instance%20status%20report%22%2C%22math%22%3A%22dau%22%2C%22name%22%3A%22instance%20status%20report%22%2C%22type%22%3A%22events%22%2C%22order%22%3A2%2C%22properties%22%3A%5B%7B%22key%22%3A%22posthog_version%22%2C%22type%22%3A%22event%22%2C%22value%22%3A%221.3.*%22%2C%22operator%22%3A%22not_regex%22%7D%2C%7B%22key%22%3A%22posthog_version%22%2C%22type%22%3A%22event%22%2C%22value%22%3A%221.2.*%22%2C%22operator%22%3A%22not_regex%22%7D%5D%2C%22custom_name%22%3A%22%3C%201.20%22%7D%5D&actions=%5B%5D&display=ActionsLineGraph&interval=week&date_from=-14d&exclusions=%5B%5D&properties=%5B%7B%22key%22%3A%22realm%22%2C%22type%22%3A%22event%22%2C%22value%22%3A%5B%22cloud%22%5D%2C%22operator%22%3A%22is_not%22%7D%5D&funnel_to_step=2&funnel_viz_type=steps&funnel_from_step=0&filter_test_accounts=true&date_to 

for <1.20 ; 1.2* vs 1.3* version usage + a more granular split for 1.3+ versions

https://app.posthog.com/insights/QEsYvzvb?insight=TRENDS&events=%5B%7B%22id%22%3A%22organization%20usage%20report%22%2C%22math%22%3A%22dau%22%2C%22name%22%3A%22organization%20usage%20report%22%2C%22type%22%3A%22events%22%2C%22order%22%3A0%2C%22properties%22%3A%5B%7B%22key%22%3A%22posthog_version%22%2C%22type%22%3A%22event%22%2C%22value%22%3A%5B%221.29.0%22%2C%221.29.1%22%5D%2C%22operator%22%3A%22is_not%22%7D%5D%2C%22custom_name%22%3A%22self-hosted%20unique%20users%22%7D%5D&actions=%5B%5D&display=ActionsLineGraph&interval=day&breakdown=posthog_version&date_from=-14d&new_entity=%5B%5D&properties=%5B%7B%22key%22%3A%22realm%22%2C%22type%22%3A%22event%22%2C%22value%22%3A%5B%22cloud%22%5D%2C%22operator%22%3A%22is_not%22%7D%5D&breakdown_type=event&filter_test_accounts=true&date_to

Reply:
Are these on the platform dash?

Reply:
yes ... we'll one of them I couldn't add properly due to an issue https://github.com/PostHog/posthog/issues/8076